packages/riipen-ui/src/components/TableHeader.test.jsx

```diff
@@ -0,0 +1,88 @@
+import { mount } from "enzyme";
+import toJson from "enzyme-to-json";
+import React from "react";
+
+import Table from "./Table.jsx";
+import TableHeader from "./TableHeader.jsx";
+import TableRow from "./TableRow.jsx";
+
+describe("<TableHeader>", () => {
+  it("renders without errors", () => {
+    let error;
+
+    try {
+      mount(
+        <Table>
+          <TableHeader />
+        </Table>
+      );
+    } catch (theError) {
+      error = theError;
+    }
+
+    expect(error).toEqual(undefined);
+  });
+
+  it("renders correct snapshot", () => {
+    const wrapper = mount(
+      <Table>
+        <TableHeader />
+      </Table>
+    );
+
+    expect(toJson(wrapper)).toMatchSnapshot();
+  });
+
+  describe("children prop", () => {
+    it("renders given non null children", () => {
+      const child = <TableRow />;
+
+      const wrapper = mount(
+        <Table>
+          <TableHeader>
+            {null}
+            {child}
+          </TableHeader>
+        </Table>
+      );
+
+      expect(
+        wrapper
+          .find("thead")
+          .childAt(0)
+          .matchesElement(child)
+      ).toEqual(true);
+    });
+  });
+
+  describe("classes prop", () => {
+    it("applies classes to the root node of Typography", () => {
```

Comment:
Should this just be "applies classes to the root node" since Typography isn't referenced in the tests?

Reply:
oooops